03-rnaseq/pathway-analysis_rnaseq_01_ora.Rmd

```diff
@@ -0,0 +1,385 @@
+Samples were collected at two time points: during their first, acute bronchiolitis visit (abbreviated "AV") and their recovery, their post-convalescence visit (abbreviated "CV").
+
+We used this dataset, to identify co-expressed genes in an [example analysis](https://alexslemonade.github.io/refinebio-examples/04-advanced-topics/network-analysis_rnaseq_01_wgcna.html) using [`WGCNA`](https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/Tutorials/) [@Langfelder2008].
+
+We have provided this file for you and the code in this notebook will read in the results that are stored online, but if you'd like to follow the steps for obtaining this results file yourself, we suggest going through [that WGCNA example](https://alexslemonade.github.io/refinebio-examples/04-advanced-topics/network-analysis_rnaseq_01_wgcna.html).
+
+In this example, we will focus on one gene module we identified through WGCNA, called module 19, to see what pathways these genes in this module may be associated with. 
+We identified this module 19 as the most differentially expressed between the datasets' two time points (during illness and recovering from illness).
+
+Here's a heatmap to show a preview of this module and the genes that make it up. 
+
+<img src="https://refinebio-examples.s3.us-east-2.amazonaws.com/04-advanced-topics/results/SRP140558_module_19_heatmap.png" width = 500/>
+
+Each row is a gene that is a member of module 19, and the composite expression of these genes, called an eigengene, is shown in the barplot below. 
+This plot demonstrates how these genes, together as a module, are differentially expressed between the two time points. 
+
+**IDENTICAL TO MICROARRAY EXAMPLE**
+## Check out our file structure!
+
+Your new analysis folder should contain:
+
+- The example analysis `.Rmd` you downloaded  
+- A folder called `data` (currently empty)
+- A folder for `plots` (currently empty)  
+- A folder for `results` (currently empty)  
+
+Your example analysis folder should contain your `.Rmd` and three empty folders (which won't be empty for long!).
+
+If the concept of a "file path" is unfamiliar to you; we recommend taking a look at our [section about file paths](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#an-important-note-about-file-paths-and-Rmds).
+
+**REVIEW**
+# Using a different refine.bio dataset with this analysis?
+
+If you'd like to adapt an example analysis to use a different dataset from [refine.bio](https://www.refine.bio/), we recommend replacing the `gene_module_url` with a different file path to a gene list that you are interested in. 
+The file we use here has two columns from our WGCNA module results, so if your gene list differs, many steps will need to be changed or deleted entirely depending on what your gene list file looks like (particularly in the [`Determine our genes of interest list` section](#determined-our-genes-of-interest-list)).
```

Comment:
Here I am trying to be a bit more explicit about what the input file contains, so people have a bit more of a sense of what the minimal data required is. It is a bit repetitive with later information, but I think it is useful to include here.

```suggestion
The file we use here has two columns from our WGCNA module results: the id of each gene and the module it is part of. 
If you'd like to adapt an example analysis to use a different dataset from [refine.bio](https://www.refine.bio/), we recommend replacing the `gene_module_url` with a different file path to a read in a similar table of genes with the information that you are interested in. 
If your gene table differs, many steps will need to be changed or deleted entirely depending on the contents of that file (particularly in the [`Determine our genes of interest list` section](#determined-our-genes-of-interest-list)).
```